### ISSUE TITLE
Error in ErrorHandler

### ISSUE THREAD

**Issue by thomthom** [thomthom/cleanup#20]
https://github.com/thomthom/cleanup/blob/9490166424c192526e567af6af1ec54970228043/src/tt_cleanup/vendor/error-handler/error_reporter.rb#L476-L488

Referring to `error` variable, should be `exception`

**Comment by thomthom** [comment]
```
Something when wrong trying to load C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/TT_CleanUp/vendor/error-handler/error_reporter.rb.
Please copy this message and forward it to the developer.
File load error (C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup/bootstrap.rb): 


Error: 
#<NameError: undefined local variable or method `error' for main:Object>
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/TT_CleanUp/vendor/error-handler/error_reporter.rb:483:in `rescue in <top (required)>'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/TT_CleanUp/vendor/error-handler/error_reporter.rb:12:in `<top (required)>'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup/bootstrap.rb:8:in `require'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup/bootstrap.rb:8:in `<top (required)>'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/extensions.rb:197:in `require'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/extensions.rb:197:in `load'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup.rb:42:in `register_extension'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup.rb:42:in `<module:CleanUp>'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup.rb:15:in `<module:Plugins>'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup.rb:14:in `<module:TT>'
C:/Users/mauro/AppData/Roaming/SketchUp/SketchUp 2023/SketchUp/Plugins/tt_cleanup.rb:13:in `<top (required)>'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/sketchup.rb:147:in `require'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/sketchup.rb:147:in `block in require_all'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/sketchup.rb:147:in `each'
C:/Users/mauro/work/sketchup/client/sketchup/products/Debug_x64/bin/sketchup_kdab/Tools/sketchup.rb:147:in `require_all'
<main>:in `<main>'
SketchUp:in `eval
```

**Comment by thomthom** [comment]
Fixed in 2217969aa2126628bd33ebc03db0ef0946f12442
